diagnostic.md

```diff
@@ -32,23 +39,30 @@ Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
 <!-- your answer starts here -->
-
+Getter Methods.
+$('div')
+$('body').children()
+Setter Methods.
+$('div').addClass("myNewClass")
+$('body').text("mytext")
 <!-- your answer ends here -->
```

Comment:
Your setter methods are correct, but getter methods are used for getting values inside the `div` and `body` elements. Think of `.val(), .html(), .text()`